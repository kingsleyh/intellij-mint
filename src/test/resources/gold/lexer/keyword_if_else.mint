module A {

  fun woop(a : Number) : Number {
    if(a == 1){
      1
    } else {
      2
    }
  }

}