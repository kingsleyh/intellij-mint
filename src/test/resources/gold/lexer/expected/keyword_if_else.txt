module ('module')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('woop')
( ('(')
IDENT ('a')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Number')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Number')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
if ('if')
( ('(')
IDENT ('a')
WHITE_SPACE (' ')
== ('==')
WHITE_SPACE (' ')
NUMBER_LITERAL ('1')
) (')')
{ ('{')
WHITE_SPACE ('\n      ')
NUMBER_LITERAL ('1')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE (' ')
else ('else')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
NUMBER_LITERAL ('2')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')